pkg/storaged/lvol-tabs.jsx

```diff
@@ -396,6 +525,117 @@ function lvol_shrink(client, lvol, info, to_fit) {
     });
 }
 
+const StructureDescription = ({ client, lvol }) => {
+    const struct = lvol.Structure;
+
+    if (!struct)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230918-080053-f89e42e4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)